packages/types/src/documents/app/automation/automation.ts

```diff
@@ -124,6 +124,7 @@ export interface Automation extends Document {
   definition: {
     steps: AutomationStep[]
     trigger: AutomationTrigger
+    stepNames?: Record<string, string>
```

Comment:
What is this a map of? Would be good to comment to explain what it does.

Reply:
{
    "0DepA21qN": "My Step Name"
}

Key is the ID of the step. For easy lookup. I'll get a comment in for it.